File: szxy-funplay-parent/funplay-sellergoods-interface/src/main/java/com/funplay/sellergoods/service/TypeTemplateService.java
package com.funplay.sellergoods.service;


import com.funplay.pojo.TypeTemplate;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface TypeTemplateService {

	PageResult findPage(TypeTemplate typeTemplate, int pageNum, int pageSize);

	TypeTemplate findOne(Long id);

	void delete(Long[] ids);

	void add(TypeTemplate typeTemplate);

	void update(TypeTemplate typeTemplate);

	List<Map> findSpeciList(Long id);

	List<TypeTemplate> findAll();

}

File: szxy-funplay-parent/funplay-manager-web/src/main/webapp/js/service/typeTemplateService.js
//服务层
app.service('typeTemplateService',function($http){


	//查询实体
	this.findOne=function(id){
		return $http.get('../typeTemplate/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../typeTemplate/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../typeTemplate/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../typeTemplate/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../typeTemplate/search.do?page='+page+"&rows="+rows, searchEntity);
	}

    //返回规格列表
    this.findSpecList=function (id) {
        return $http.get('../typeTemplate/findSpecList.do?id='+id);
    }

});
